src/annotator/integrations/vitalsource.js

```diff
@@ -146,6 +146,38 @@ function getPDFPageImage() {
   );
 }
 
+/**
+ * Fix how a VitalSource book content frame scrolls, so that various related
+ * Hypothesis behaviors (the bucket bar, scrolling annotations into view) work
+ * as intended.
+ *
+ * Some VitalSource books (PDFs) make content scrolling work by making the
+ * content iframe really tall and having the parent frame scroll. This stops the
+ * Hypothesis bucket bar and scrolling annotations into view from working.
+ *
+ * @param {HTMLIFrameElement} frame
+ */
+function makeContentFrameScrollable(frame) {
```

Comment:
Good function name 👍🏻 This is exactly where too much abstraction could be detrimental...